Added "server" parameter description in CloudHub deployment article.

Cristian-Venticinque opened the pull request:
Added "server" parameter description in CloudHub deployment article.

Commit:
Added "server" parameter description in CloudHub deployment article.

Added "server" parameter description in CloudHub deployment article.

dukesphere commented on modules/ROOT/pages/cloudhub-deployment-mmp-reference.adoc:
```diff
@@ -66,8 +66,8 @@ Accepted values are:
-//_TODO: validate this
-// | server | Maven server with Anypoint Platform credentials. This is only needed if you want to use your credentials stored in your Maven settings.xml file. This is not the Mule server name.
+
+| server | Maven server with Anypoint Platform credentials. This is only needed if you want to use your credentials stored in your Maven settings.xml file. This is not the Mule server name.
```
style tip: please put backticks around file names, e.g.: 
settings.xml -> `settings.xml` 
to change the font

Commit:
Styling update. 

-All file names mentioned in the MMP article  are properly styled now.
-Removed (Mule Maven Plugin) from title names to be consistent with the 
other articles and to improve readability.

Commit:
Added 'server' parameter description in additional articles.

-The Server parameter was also missing in the reference page of Runtime 
Fabric and ARM, added to these articles as well. 
-Minor styling update on titles.
